package.json

```diff
@@ -5,14 +5,19 @@
   "main": "./build/main.js",
   "scripts": {
     "build": "roamjs-scripts build",
-    "dev": "roamjs-scripts dev"
+    "dev": "roamjs-scripts dev",
+    "test": "TZ=UTC jest --config jest.config.ts"
```

Comment:
can we `npm install --save-dev cross-env` and put it at the beginning here? this command won't work on windows by default. Here's an example:

https://github.com/dvargas92495/roamjs-static-site/blob/main/package.json#L9

Reply:
Ah! windows, yeah, that... 🤦 
Will do!